openedx/core/lib/xblock_utils/__init__.py

```diff
@@ -412,7 +412,12 @@ def add_staff_markup(user, disable_staff_debug_info, block, view, frag, context)
         'disable_staff_debug_info': disable_staff_debug_info,
     }
     if isinstance(block, ScorableXBlockMixin):
-        staff_context['max_problem_score'] = block.max_score()
+        try:
```

Comment:
Why `try/except` is required given that the code before `block.max_score()` wasn't part of it?

Reply:
`max_score` function is instantiating a LoncapaProblem inside, which in turn calls the `make_xml_compatible` function and results in exception due to being multiple correct answers defined for dropdown.

Reply:
got it.